tools/lipsyncthing/main.cpp

```diff
@@ -1,11 +1,10 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include "stdint.h"
 #include <cstring>
-#include <direct.h>
-#include <io.h>
-#include <windows.h>
 
+#define byte uint8_t
```

Comment:
I would just typedef unsigned char byte like the main source tree.

And use <> for stdint.h.  May need the wrapper like q_shared.h too for MSVC older versions etc.